msteams-platform/m365-apps/actions.md

```diff
@@ -0,0 +1,150 @@
+---
+title: Actions in Microsoft 365
+description: In this article, learn more about the function of Actions and its use cases. 
+ms.date: 11/12/2023
+ms.author: mosdevdocs
+author: mobajemu
+ms.topic: Conceptual
+ms.subservice: m365apps
+---
+# Actions in Microsoft 365
+
+> [!NOTE]
+> The Actions in Microsoft 365 is only available for Microsoft 365 app on web and desktop clients.
+> The Actions in Microsoft 365 is available in public developer preview.
+
+Actions seamlessly integrate your app into the user's workflow, ensuring effortless discovery and smooth interaction with their content. By guiding users to your app based on their intent and contextual content, Actions facilitate efficient task completion. Which integration increases your app's visibility and engagement with minimal development effort.
```

Comment:
"seamlessly" and "effortless" might be overselling this a bit...suggest dropping those adjectives. Also "Which integration increases your app's visibility and engagement with minimal development effort." . . . I don't think that's a sentence?